### PR TITLE
Added ci tests for adapter  

### PR DESCRIPTION
**Pull request by Revolyssup**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Revolyssup** [comment]
https://github.com/Revolyssup/meshery-istio/runs/4299386220?check_suite_focus=true 
The tests are passing on my master. Here the meshery server panics maybe due to bad/invalid token. Just have to confirm what the key for the token is

**Comment by leecalcote** [comment]
This repo says that the PROVIDER_TOKEN was updated an hour before this PR was submitted...

**Comment by DelusionalOptimist** [comment]
@Revolyssup the panic seems to be arising because the token is empty:
see:
https://github.com/meshery/meshery-istio/runs/4299386943?check_suite_focus=true#step:5:6
https://github.com/meshery/meshery-istio/runs/4299386943?check_suite_focus=true#step:5:9

**Comment by Revolyssup** [comment]
> 

https://github.com/Revolyssup/meshery-istio/runs/4300310895?check_suite_focus=true#step:5:6
And in my repo its not empty, it means most probably the correct secret is not I guess, or set by some other key

**Comment by DelusionalOptimist** [comment]
This doesn't imply that the secret itself is empty but maybe github isn't giving it to the workflow or it might be getting lost in bash.

File: .github/workflows/testistioadapter.yaml [hunk 1]
```diff
@@ -0,0 +1,130 @@
+name: Test Istio Adapter
+
+on:
+  push:
+    branches:
+      - "*"
+    tags:
+      - "v*"
+  pull_request:
```
Comment (DelusionalOptimist) [inline .github/workflows/testistioadapter.yaml:9]: @Revolyssup try changing this to `pull_request_target`
https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request_target

**Comment by Revolyssup** [comment]
@MarioArriaga92

**Pull request closed without merging**